haskell-tests/Types/CBOR.hs

```diff
@@ -153,26 +171,26 @@ testInitializationParameters = describe "token-initialization-parameters decodin
             ( deserialiseFromBytes
                 decodeTokenInitializationParameters
                 "\xA4\x64name\x69\
-                \ABC token\x68metadata\x76https://abc.token/meta\x69\
+                \ABC token\x68metadata\xA1\x63url\x76https://abc.token/meta\x69\
                 \allowList\xF4\x6DinitialSupply\xC4\x82\x24\x19\x27\x10"
             )
     it "Missing \"name\"" $
         assertEqual
             "Decoded CBOR"
-            (Left (DeserialiseFailure 64 "Missing \"name\""))
+            (Left (DeserialiseFailure 69 "Missing \"name\""))
```

Comment:
Byte offset changes by 5 bytes due to the changes to the 5 bytes added `"\xA1\x63url"` to the cbor representation of the metadata url